infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/actions/StateTransfer.java

```diff
@@ -42,6 +44,13 @@ private StateTransfer() {
     // Maximum number of retries after which the Overwrite Exception is rethrown.
     private static final int OVERWRITE_RETRIES = 3;
 
+    // Default read options for the state read calls
+    private static ReadOptions readOptions = ReadOptions.builder()
+            .waitForHole(true)
```

Comment:
I think it would be safe to skip waiting for hole fill on state transfer?

Reply:
We can experiment with it in a different PR. This PR, is mostly code shuffling, don't want to change configurations.